spec/did-resolution.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
```

Comment:
idk if we conform to this now
https://github.com/TBD54566975/web5-spec/blob/main/test-vectors/did_dht/resolve.json

Reply:
yeah DHT is not supporting this since we don't have a dht resolver yet 

https://github.com/TBD54566975/did-dht-method/issues/18